contractcourt/chain_watcher.go

```diff
@@ -393,6 +393,96 @@ func isOurCommitment(localChanCfg, remoteChanCfg channeldb.ChannelConfig,
 	return false, nil
 }
 
+// chainSet includes all the information we need to dispatch a channel close
+// event to any subscribers.
+type chainSet struct {
+	// remoteStateNum is the commitment number of the lowest valid
+	// commitment the remote party holds from our PoV.
+	remoteStateNum uint64
+
+	// commitSet includes information pertaining to the set of active HTLCs
+	// on each commitment.
+	commitSet CommitSet
+
+	// remoteCommit points to the current commitment of the remote party.
+	remoteCommit *channeldb.ChannelCommitment
```

Comment:
make `remoteCommit` and `localCommit` non-pointers, to make it clear they are not optional?